### DISCUSSION TITLE
Unhandled Runtime Error / next-firebase-auth

### DISCUSSION THREAD

**Discussion by uancamilo** [gladly-team/next-firebase-auth#554] [Q&A] [answered]
Hello.. 

I'm trying to build something with an auth component using next-firebase-auth but is giving me the same error. 

if I have followed the instructions in the README.MD why do I have this error message? 

![Captura desde 2022-09-10 23-47-15](https://user-images.githubusercontent.com/36907625/189513337-a69a4be1-f3ba-4118-bb39-257558bb38cc.png)

Thanks for any help whit this.

**Comment by kmjennison** [comment, marked as answer]
You might be using incompatible version of Firebase and `next-firebase-auth`. For Firebase v9+, be sure you're using `next-firebase-auth@canary`.

**Reply by uancamilo** [reply]
Yes, and thanks and apology for this silly question...  is working perfectly with this change. my bad. for not reading carefully.
